bossdata/meta.py

```diff
@@ -98,36 +101,68 @@ def create_meta_lite(sp_all_path, db_path, verbose=True):
             to be a gzipped ASCII data file.
         db_path(str): Local path where the corresponding sqlite3 database will be written.
     """
+    
+    #parameters controlling how parse/inserts are chunked
+    chunk_size_str = os.getenv('BOSS_CHUNK_SIZE')
```

Comment:
I would like to avoid adding another envvar if possible, especially for an implementation parameter (rather than a user config parameter).  Will a fixed value work, or can you code the logic for setting a sensible value?